pkg/install/templates/tiller-ca-cert-configmap.yaml.tmpl

```diff
@@ -0,0 +1,11 @@
+{{ if .EnableTillerTLS }}---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: flux-helm-tls-ca-config
+  {{ if .Namespace }}
+  namespace: {{ .Namespace }}{{ end }}
+data:
+  ca.crt: |
+    {{ Base64Encode .TillerTLSCACertContent }}
+{{- end -}}
```

Comment:
Newline

Reply:
It's intentional, so that the file is empty when no TLS is enabled (to discard it). I can change the length check to `<= 1` though